setup.py

```diff
@@ -40,7 +40,7 @@
     },
     classifiers=[
         "Framework :: Django",
-        "Framework :: Django :: 2",
+        "Framework :: Django :: 3",
```

Comment:
This should be an addition, not a replacement.

```suggestion
        "Framework :: Django :: 2",
        "Framework :: Django :: 3",
```

Reply:
Good catch. This change has been made.